cli/src/commands/init.rs

```diff
@@ -98,23 +103,53 @@ fn prompt_project_name() -> io::Result<String> {
     let default_name = current_dir.file_name().and_then(|name| name.to_str());
 
     let mut prompt = Input::new();
-    prompt.with_prompt("Project Name");
 
-    if let Some(default_name) = default_name {
-        prompt.default(default_name.to_owned());
+    // Suggest default if we found a directory name.
+    //
+    // NOTE: We don't use dialoguer's built-in default here so we can add a more
+    // explicit `default` label.
+    match default_name {
+        Some(default_name) => {
+            prompt.with_prompt(format!("Project Name [default: {default_name}]"));
+            prompt.allow_empty(true);
+        },
+        None => {
+            let _ = prompt.with_prompt("Project Name");
+        },
+    };
+
+    let mut name: String = prompt.interact_text()?;
+
+    // Fallback to project name for empty strings.
+    if name.is_empty() {
+        name = default_name.expect("illegal empty project name").into();
     }
 
-    prompt.interact_text()
+    Ok(name)
 }
 
 /// Ask for the desired group.
-fn prompt_group() -> io::Result<Option<String>> {
-    let group: String = Input::new()
-        .with_prompt("Project Group [default: no group]")
-        .allow_empty(true)
-        .interact_text()?;
+async fn prompt_group(groups: &[UserGroup]) -> anyhow::Result<Option<String>> {
+    // Skip group selection if user has none.
+    if groups.is_empty() {
+        return Ok(None);
+    }
+
+    // Map groups to their name.
+    let group_names = iter::once("[None]")
+        .chain(groups.iter().map(|group| group.group_name.as_str()))
+        .collect::<Vec<_>>();
+
+    println!();
 
-    Ok((!group.is_empty()).then_some(group))
+    // Prompt for selection of one group.
+    let prompt = "[ENTER] Select and Confirm\nProject Group";
```

Comment:
I'd argue this is **less** clear. I explicitly chose brevity here because otherwise this can not be immediately identified as keybinding instructions.

Reply:
These decisions are tough. We might need to hire a human interfaces expert (I think they call them UX engineers now). The suggestion here could be overcome with the inclusion of a "top-level" instruction/usage statement about how selection and defaults work.

Reply:
Group decided to keep this one as is.